tests/test_cases/test_dumpfile_verilator/Makefile

```diff
@@ -0,0 +1,24 @@
+# Copyright cocotb contributors
+# Copyright (c) 2015 Potential Ventures Ltd
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+TOPLEVEL_LANG ?= verilog
+SIM ?= verilator
+EXTRA_ARGS += --trace
+
+ifneq ($(TOPLEVEL_LANG),verilog)
```

Comment:
I'd switch this to checking if SIM is verilator. The selected `EXTRA_ARGS` only work on it.